pubspec.yaml

```diff
@@ -3,14 +3,14 @@
 # license that can be found in the LICENSE file.
 
 name: codemirror
-version: 0.5.24+5.59.2
+version: 0.6.0+5.60.0
```

Comment:
Do you want to publish as a dev release first, in order to test it out in some clients - like DartPad - or publish as a stable?

Reply:
Dev release sounds like a good idea